docs/xxdiagnosesynconvaluebasedclasses.md

```diff
@@ -0,0 +1,40 @@
+<!--
+* Copyright (c) 2021, 2021 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# -XX:DiagnoseSyncOnValueBasedClasses
+
+![Start of content that applies only to Java 16 and later](cr/java16plus.png) This option provides warnings about improper attempts to synchronize on instances of a value-based class. ![End of content that applies only to Java 16 or later](cr/java_close.png)
+
+
+## Syntax
+
+        -XX:DiagnoseSyncOnValueBasedClasses=1
```

Comment:
For consistency with the rest of the topics, please can we use the format:

```
-XX:DiagnoseSyncOnValueBasedClasses=<number>
```
Where `<number>` is one of the following values:

- `1`: OpenJ9 throws a `VirtualMachineError` on improper attempts to synchronize on instances of value-based classes.
- `2`: OpenJ9 prints ~~out~~ a warning message on improper attempts to synchronize on instances of value-based classes.

Reply:
Is there a default?

Reply:
> Is there a default?

No, there is no default. User has to specify either 1 or 2 to `<number>`

Reply:
Updated according to the review comment.